packaging/windows/clisdk/VS.Redist.Common.Net.Core.SDK.x64.nuspec

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think this nuspec is required for generating the MSBuildExtensions insertion package.